### ISSUE TITLE
Audio single result page keyboard navigation is broken

### ISSUE THREAD

**Issue by sarayourfriend** [WordPress/openverse-frontend#623]
## Description

It is not possible to tab out of the waveform component on the audio result page, nor does the waveform respond to keyboard events anymore for seeking or play/pause.

## Reproduction

1. Run `main` and go to http://localhost:8443/audio/632d0076-fdbd-4919-8b8f-b498ef246ee8
2. Use `tab` to reach the waveform
3. Try to "tab out" of the waveform by hitting tab again.
4. Try to seek the audio using your arrow keys.
5. Try to toggle the audio playback using the spacebar.

Note that none of 3-5 will be successful.

## Expectation

All of 3-5 should work. You should be able to tab out of the audio waveform, toggle the playback state, and seek, all from the keyboard.

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.
